packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -597,6 +600,22 @@ export class DashboardModel {
         return this.getById(dashboardUuid);
     }
 
+    async updatePinning(
+        dashboardUuid: string,
+        dashboard: DashboardUnversionedFields,
```

Comment:
Since this method is just about updating pinning we can make this argument type stricter. 

ps: Same for the chart method